recipes/r-hillr/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set version = '0.5.1' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-hillr
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/hillR_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/hillR/hillR_{{ version }}.tar.gz
+  sha256: 36adecac41c0476af81d253bf5adf97e4f33cf634579a78c336322f1ff70e27b
+
+build:
+  merge_build_host: True  # [win]
+  skip: true  # [win]
```

Comment:
Why is Windows skipped here?

Reply:
If there's a good reason (e.g. dependency doesn't build on Windows etc) please add a comment above the skip with a short description.